third_party/src/default/assets/js/main.js

```diff
@@ -413,46 +413,46 @@ function sanitizeModuleName(moduleName) {
     return moduleName.toLowerCase().replace(/[^a-z0-9]/gi, '_');
 }
 function renderSimpleHTMLRecursive(obj, package, spacing) {
-    if (package === void 0) { package = ""; }
-    if (spacing === void 0) { spacing = "&emsp;"; }
-    var html = "";
+    if (package === void 0) { package = ''; }
```

Comment:
Did you change this file yourself? This is an auto-generated file.

Reply:
it is caused by `npm run build`, I think

Reply:
Okay, this file is auto-generated. Any changes you want to make should be done in the original source file. Otherwise your changes will be overridden.